pkg/server/admin.go

```diff
@@ -2603,7 +2603,17 @@ func (s *adminServer) enqueueRangeLocal(
 		return response, nil
 	}
 
-	traceSpans, processErr, err := store.ManuallyEnqueue(ctx, req.Queue, repl, req.SkipShouldQueue)
+	// Handle mixed-version clusters across the "gc" to "mvccGC" queue rename.
+	// TODO(nvanbenschoten): remove this in v23.1. Inline req.Queue again.
```

Comment:
Wow, that's long off. That's because:

- 22.1 will understand gc and mvccGC but will have to send ... gc? To interop with 21.2 nodes?
- 22.2 will send mvccGC but will have to understand gc to deal with 22.1
- 23.1 finally gets to use MVCCgc throughout?

Worth a comment imo.